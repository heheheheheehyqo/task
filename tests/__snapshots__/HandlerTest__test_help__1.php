<?php return 'Example:
--message=<string> [--number=<int>] [--flag=<bool>]

Required:
--message=<string>

Optional:
--number=<int>
--flag=<bool>
';
