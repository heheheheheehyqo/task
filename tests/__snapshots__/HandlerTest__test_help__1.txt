Example:
--message=<string> [--number=<int>] [--flag=<bool>]

Required:
--message=<string> â€” Message for return

Optional:
--number=<int>
--flag=<bool>
